### DISCUSSION TITLE
Retrieve Private Media

### DISCUSSION THREAD

**Discussion by brandon-9** [spatie/laravel-medialibrary#2677] [Q&A]
We have a Private Digital Ocean Space, and Media Library Pro is uploading images to this fine. It doesn't however display the images due to the Space being private, and just accessing the URL of the image results in an AccessDenied error.

Is there anything built-in that we're missing that connects to the space with the correct access key to properly display the image? It uploads fine using the credentials. 

Thanks!

**Comment by heidilux** [comment]
Older question, I know, but just ran into this. You need to add `'visibility' => 'public',` to your digital ocean "disk" config, in the same array as your key, secret, etc.
